libportal/remote.c

```diff
@@ -195,13 +332,17 @@ devices_selected (GDBusConnection *bus,
         select_sources (call);
       else
         {
+          XdpRemoteDesktopSession *session = g_object_new (XDP_TYPE_REMOTEDESKTOP_SESSION, NULL);
+
```

Comment:
One pattern I've been trying to adhere is to always declare GObject variables as `g_autoptr()`, and if necessary return / pass them using `g_steal_pointer()`.